libs/device/pxt.json

```diff
@@ -39,7 +39,8 @@
     ],
     "core": true,
     "dependencies": {
-        "hw": "file:../hw"
+        "hw": "file:../hw",
+        "online-multiplayer": "file:../online-multiplayer"
```

Comment:
@jwunderl, @abchatra  I wonder if online-multiplayer should be a built-in extension. That, or not a separate package at all.

Reply:
Yes, the reason I added it as a dependency here was to cause it to be packed in for share links that came from normal arcade (that is, they kept the dependency on device and this was implicitly added to the project only in my upload target).

Reply:
Would you recommend moving it to `game`?

Reply:
yes, I had originally implemented as a separate package primarily to make it easier to separate out / minimize chances of conflicts with game so that it could be dropped in directly the future with a few renamings and moving the files whenever we came back to it.

(chatted on teams about this but just leaving comment for future ref~)